x-pack/qa/full-cluster-restart/src/test/java/org/elasticsearch/xpack/restart/FullClusterRestartIT.java

```diff
@@ -390,6 +391,83 @@ public void testRollupAfterRestart() throws Exception {
         }
     }
 
+    public void testTransformLegacyTemplateCleanup() throws Exception {
+        assumeTrue("Before 7.2 transforms didn't exist", getOldClusterVersion().onOrAfter(Version.V_7_2_0));
+        if (isRunningAgainstOldCluster()) {
+
+            // create the source index
+            final Request createIndexRequest = new Request("PUT", "customers");
+            createIndexRequest.setJsonEntity(
+                "{"
+                    + "\"mappings\": {"
+                    + "  \"properties\": {"
+                    + "    \"customer_id\": { \"type\": \"keyword\" },"
+                    + "    \"price\": { \"type\": \"double\" }"
+                    + "  }"
+                    + "}"
+                    + "}"
+            );
+
+            // create a transform
```

Comment:
I think it would make sense to move lines 411-415 right before line 420 so that all the code for creating transform is in one place, not interleaved with index creation code.

Reply:
Yes, sorry, I didn't intend to interleave it.  Too much copying and pasting 😆 .